### ISSUE TITLE
modularScale throws an error if its second argument is a number

### ISSUE THREAD

**Issue by ervasive** [styled-components/polished#429]
The [documentation](https://polished.js.org/docs/#modularscale) states that the second parameter to the `modularScale` can be either a `number` or `string`, but when I try to provide a unitless number it throws an error.

- `polished` version: 3.2.0

Mixin/Helper/Shorthand you were using and how you were using it:

```javascript
modularScale(1, 1.5)
```

What you are seeing:

```
Error: Invalid value passed as base to modularScale, expected number or em/rem string but got 1.5.`
```

What you expected to see:

1.9994999999999998 (as a plain number without units)

Reproduction:

https://codesandbox.io/s/133lqrox73 (look at console output)

Thanks.

**Comment by bhough** [comment]
Thanks @ervasive 

We're working on a re-write of `modularScale` that should be out soon that will address this problem and free it from its current `em/rem` limitation.

**Comment by bhough** [comment]
Addressed in 3.3.0
